packages/shopify-codemod/transforms/global-reference-to-import.js

```diff
@@ -70,13 +70,33 @@ export default function globalReferenceToImport(
   }
 
   function findNamedExports(filename) {
-    if (extname(filename) === '.coffee') {
-      return [];
+    let namedExports = [];
+    switch (filename.slice(filename.indexOf('.'))) {
+    case '.coffee':
+      break;
+    case '.js':
+      // eslint-disable-next-line no-sync
+      namedExports = j(fs.readFileSync(filename).toString())
+        .find(j.ExportNamedDeclaration)
+        .paths();
+      break;
+    default: {
+      const absolutePath = resolve(filename);
+      const regex = '^export\\s+\\S+';
+      const result = spawnSync(binary, [...args, regex, absolutePath]);
+
+      if (result.error != null) {
+        throw result.error;
+      } else if (result.status === 1) {
+        break;
+      }
+
+      const stdout = result.stdout.toString();
+      const exportLines = stdout.trim().split('\n').map(getExportLine);
+      namedExports = filterDefaultExports(exportLines);
     }
-    // eslint-disable-next-line no-sync
-    const namedExports = j(fs.readFileSync(filename).toString())
-      .find(j.ExportNamedDeclaration);
-    return namedExports.paths();
+    }
```

Comment:
There is now a `}` right above another `}` on the same column.

This is due to the collision of 2 `eslint` rules:
 - Switch case indenting requiring `case`/`default` lines to be on the same column as the `switch`.
 - Lexical declarations within `case` statements require a `{ }` wrapper clause to prevent hoisting out of the `case`.